Adjust README for Source Code Example (#13)
GDP-ADMIN/gen-ai-examples: examples/gen-ai-internal-hello-world/README.md

> # GenAI RAGO Example using gen-ai-internal

Reviewer: ```suggestion
# GenAI RAGO Example Using gen-ai-internal
```

Author: Gw update jadi `# GenAI RAGO Example Using Source Code Version of SDK`
yg binary jadi `# GenAI RAGO Example Using Binary Version of SDK`.

wdyt?

Reviewer 2: lgtm sih

Reviewer: agree

> # GenAI RAGO Example Using Source Code Version of SDK

Reviewer 2: ```suggestion
# GenAI RAGO Example Using the Source Code Version of the SDK
```

> This is an example of how to use the gllm-pipeline library to build a simple RAG pipeline using source code version of SDK.

Reviewer 2: ```suggestion
This is an example of how to use the gllm-pipeline library to build a simple RAG pipeline using the source code version of the SDK.
```

> See other examples in [gen-ai-hello-world](https://github.com/GDP-ADMIN/gen-ai-examples).

Reviewer: should be `gen-ai-examples`?

     local-start.bat
     ```

   The program will then wait for your question:

   > Question:

   Type your question and press enter. For example, you can try asking:

   ```
   What are the documents?
   ```

   The RAG pipeline will return the following response (more or less):

   > _The documents mentioned are referred to as Mock document 1, Mock document 2, and Mock document 3. However, without additional context or content from these documents, I cannot provide specific details about their contents or purposes._

## Setting Python Interpreter Path in VSCode IDE

> [!WARNING]
> You need to have successfully run the [Running the Code](#running-the-code) part and got the response before continuing with steps in this part

If you have successfully gotten the response, open the code in VSCode IDE. Afterwards you need to set the Python interpreter path. If you haven't set the Python interpreter path, you will get error about import like below (the screenshot below is code in [main.py](/examples/gen-ai-internal-hello-world/gen_ai_internal_hello_world/main.py#L13-L20)).

![alt text](img/image-import-error.png)

> Phython interpreter path is required so the IDE can recognize the import statements. In the console, below the response, there will be this log: `Getting python interpreter path for use in IDE...` copy paste the next line, it's usually something like:

Reviewer 2: ```suggestion
Setting the Python interpreter path is necessary for the IDE to recognize the import statements. In the console, below the response, you will see the log: `Getting python interpreter path for use in IDE...` Copy and paste the next line, which usually looks something like:
```
